tests/unit/search-headless.ts

```diff
@@ -349,14 +376,26 @@ describe('search works as expected', () => {
   it('vertical search works', async () => {
     answers.state.meta.searchType = SearchTypeEnum.Vertical;
     await answers.executeVerticalQuery();
-    const { fieldId, matcher, value } = mockedState.filters.static[0];
+    const { kind, fieldId, matcher, value } = mockedState.filters.static[0].filter as FieldValueStaticFilter;
+    const {
+      kind: disjunctionKind,
+      combinator,
+      filters
+    } = mockedState.filters.static[1].filter as DisjunctionStaticFilter;
     const coreCalls = mockedCore.verticalSearch.mock.calls;
     const expectedSearchParams = {
       query: mockedState.query.input,
       querySource: mockedState.query.querySource,
       queryTrigger: mockedState.query.queryTrigger,
       verticalKey: mockedState.vertical.verticalKey,
-      staticFilters: { fieldId, matcher, value },
+      staticFilter: {
+        kind: 'conjunction',
+        combinator: FilterCombinator.AND,
+        filters: [
+          { kind, fieldId, matcher, value },
+          { kind: disjunctionKind, combinator, filters }
```

Comment:
Could this just be `mockedState.filters.static[0]` instead of { kind: disjunctionKind, combinator, filters }?